app/helpers/dashboard_helper.rb

```diff
@@ -90,4 +90,13 @@ def report_color
     }
   end
 
+  def auto_refresh_button(options = {})
+    on = options[:defaults_to] ? "on" : "off"
+    if params[:auto].present?
+      on = params[:auto] == "0" ? "off" : "on"
+    end
+    link = link_to("<i class='glyphicon glyphicon-refresh'></i>".html_safe, {:auto => (on == "on" ? "0" : nil)}, { :'data-original-title' => "Auto refresh #{on}", :rel => 'twipsy' })
```

Comment:
extracted to i18n, don't think I understand what you mean by on/off to two strings...